front-end/Reserva_facil/src/app/Componets/room-component/room-component.component.ts

```
@@ -0,0 +1,41 @@
+
+@Component({
+  selector: 'app-room-component',
+  templateUrl: './room-component.component.html',
+  standalone: true,
+  imports: [RoomCardComponent, CommonModule]
+})
+export class RoomComponent implements OnInit {
+  rooms: Room[] = [];
+  loading = true;
+
+  constructor(private roomService: RoomService, @Inject(PLATFORM_ID) private platformId: Object) {}
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)